retry/lib/retry.dart

```diff
@@ -110,9 +111,9 @@ class RetryOptions {
   }
 
   /// Call [fn] retrying so long as [retryIf] return `true` for the exception
-  /// thrown.
+  /// thrown, or if [retryIfValue] returns `true` for the value returned from [fn].
   ///
-  /// At every retry the [onRetry] function will be called (if given). The
+  /// At every retry the [onRetry] and [onRetryIfValue] functions will be called (if given). The
```

Comment:
Shouldn't be an OR, either `onRetry` is called or `onRetryValue` is called?

And don't we need to explain which is called when...